app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h3>Contestants</h3>
+
+<% @contestants.each do |contestant| %>
+  <%= contestant.name %>
+  <P>Projects: 
+  <% contestant.projects.each do|project| %>
+   <%= project.name%> , </p>
```

Comment:
You can do a nested iteration like you have but you could also build a method in your `Contestant` model that grabs all of the projects names. So something like:
```ruby
def projects_list
  projects.pluck(:name).to_sentence
end
```
So line 5 would read:
```ruby
<p>Projects: <%= contestant.projects_list %></p>
```
and get rid of the rest.